compiler-plugin/src/main/java/io/ballerina/stdlib/persist/compiler/PersistRecordValidator.java

```diff
@@ -180,15 +174,13 @@ private Optional<AnnotationNode> getEntityAnnotation(NodeList<AnnotationNode> an
         return Optional.empty();
     }
 
-    private boolean isDuplicateEntity(Entity entity, TypeDefinitionNode typeDefinitionNode) {
+    private void addWarningIfDuplicateEntity(Entity entity, TypeDefinitionNode typeDefinitionNode) {
```

Comment:
This is not a warning to duplicate entities. But a warning when users have defined entities in multiple modules. Take a look at the issue again.

Reply:
Updated in [2b9d986](https://github.com/ballerina-platform/module-ballerina-persist/pull/87/commits/2b9d986e61cfef61f255c9508e2b5a903c70a5cd)

Reply:
You are still checking for duplicate entities. As I said lets have a call offline to figure this out.

Reply:
Updated in [66754da](https://github.com/ballerina-platform/module-ballerina-persist/pull/87/commits/66754daa6449de177f60854f1971cedc11336751)